### ISSUE TITLE
Hi!

### ISSUE THREAD

**Issue by MarianoGenco** [ThomasDepole/wordpress-child-plugin-tool#2]
Nice Try! Is great the tool!

I have a question about it.
What happens if you update the main plugin and for some reason there is any part that changes in the line of code?

Then by putting apply.php it will put the modified files with the risk that some file is not the same as the main plugin due to its update.

I feel that the tool has potential, I even think of adding some things to it.

I only have this doubt that the main plugin is updated and just change a file that it modifies.

The second question if after apply.php I have to delete everything and only leave the modifiers folder?

Basically then it only replaces modified files that I can save on the hosting automatically.

It could be added to make a reading of the previous original file and the original file after the update. If the 2 files are the same just there, execute the changes (Apply.php) Otherwise, do not do it and warn that the new file in the updated plugin has been changed by it.
